examples/README.md

```diff
@@ -20,6 +20,7 @@ Container engines:
 - [`podman.yaml`](./podman.yaml): Podman
 - [`singularity.yaml`](./singularity.yaml): Singularity
 - LXD is installed in the default Ubuntu template, so there is no `lxd.yaml`
+- [`buildkit.yaml`](./buildkit.yaml): Buildkit
```

Comment:
I'm not sure BuildKit can be categorized as "Container engines".
Maybe "Container image builders"?

Reply:
Sure